include/swift/AST/IRGenOptions.h

```diff
@@ -67,6 +68,9 @@ class IRGenOptions {
   std::vector<std::string> OutputFilenames;
   std::string ModuleName;
 
+  /// \brief User-overridable language version to compile for.
+  version::Version EffectiveLanguageVersion = version::Version::getCurrentLanguageVersion();
+
```

Comment:
Ah, thinko -- I was looking at IRGenOptions and for some reason didn't notice IRGenModule was also in scope for this usage.